[2024-03-29T17:01:54.006+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_templating.t5 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:54.110+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_templating.t5 manual__2024-03-29T17:01:50.584301+00:00 [queued]>
[2024-03-29T17:01:54.120+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-29T17:01:54.225+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): t5> on 2024-03-29 17:01:50.584301+00:00
[2024-03-29T17:01:54.304+0000] {standard_task_runner.py:60} INFO - Started process 3244 to run task
[2024-03-29T17:01:54.368+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'demo_templating', 't5', 'manual__2024-03-29T17:01:50.584301+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/demo_templating.py', '--cfg-path', '/tmp/tmpigqghl4b']
[2024-03-29T17:01:54.479+0000] {standard_task_runner.py:88} INFO - Job 48: Subtask t5
[2024-03-29T17:01:54.740+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-29T17:01:55.210+0000] {task_command.py:423} INFO - Running <TaskInstance: demo_templating.t5 manual__2024-03-29T17:01:50.584301+00:00 [running]> on host c4cb237119ff
[2024-03-29T17:01:56.968+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='domwatcharin' AIRFLOW_CTX_DAG_ID='demo_templating' AIRFLOW_CTX_TASK_ID='t5' AIRFLOW_CTX_EXECUTION_DATE='2024-03-29T17:01:50.584301+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-03-29T17:01:50.584301+00:00'
[2024-03-29T17:01:56.978+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-03-29T17:01:56.999+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo 2022-02-11']
[2024-03-29T17:01:57.152+0000] {subprocess.py:86} INFO - Output:
[2024-03-29T17:01:57.647+0000] {subprocess.py:93} INFO - 2022-02-11
[2024-03-29T17:01:57.682+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-03-29T17:01:58.389+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=demo_templating, task_id=t5, execution_date=20240329T170150, start_date=20240329T170154, end_date=20240329T170158
[2024-03-29T17:02:00.670+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-29T17:02:01.438+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
